cmd/tableprinters/machine.go

```diff
@@ -18,7 +18,7 @@ func (t *TablePrinter) MachineTable(data []*models.V1MachineResponse, wide bool)
 		rows [][]string
 	)
 
-	header := []string{"ID", "", "Last Event", "When", "Age", "Hostname", "Project", "Size", "Image", "Partition"}
+	header := []string{"ID", "", "Last Event", "When", "Age", "Hostname", "Project", "Size", "Image", "Partition", "Rack"}
```

Comment:
IMHO wide should always contain all the fields from the normal output + additional fields.

Reply:
will do

Reply:
Done

Reply:
For the IPMI printer I can see it, but here it is still missing?

Reply:
stupid me

Reply:
done